### ISSUE TITLE
[Feature request] When importing same app / duplicating, change app name also

### ISSUE THREAD

**Issue by eXistenZe** [2sic/2sxc#1606]
**I'm submitting a ...**
[x] feature request

**...about**
[x] admin experience UI

**Current behavior**
When importing the same app, it will duplicate it and ask to change the folder name. App name will be the same.

**Expected behavior**
App name should be changed too, or else you are forced to open app settings, check folder name, change app name to make sure which app you're dealing with.

**Comment by iJungleboy** [comment]
Good idea, but ATM not on our priorities. Would need a contribution or sponsor, my estimate is with create and testing to be about 2h.
